clang/lib/Driver/Driver.cpp

```diff
@@ -4791,6 +4791,20 @@ void Driver::BuildActions(Compilation &C, DerivedArgList &Args,
     if (!UnbundlerInputs.empty()) {
       Action *PartialLink =
           C.MakeAction<PartialLinkJobAction>(UnbundlerInputs, types::TY_Object);
+      if (!LastArg) {
+        auto *tAC = dyn_cast<Action>(UnbundlerInputs.back());
```

Comment:
What does `tAC` stand for? A more obvious naming could be beneficial. Or, in case I'm just missing a common abbreviation, this [naming standard](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly) tweak would be necessary:
```suggestion
        auto *TAC = dyn_cast<Action>(UnbundlerInputs.back());
```

Reply:
I just meant it to be a temporary action abbreviation.  I can make it more clear and all capitals as suggested.